src/test/unit/test.js

```diff
@@ -384,25 +384,37 @@ describe('Client', function() {
       })
```

Comment:
you can remove the changes to the `test.js` we are going to remove all the mock tests in test.js

Unit tests will test only functions which do not interact with external world (i.e which make calls to S3)